src/test/mocks.ts

```diff
@@ -0,0 +1,429 @@
+import { CategoryEntity } from '../category/infrastructure/persistence/relational/entities/categories.entity';
```

Comment:
@codextech overwriting the whole mocks file is nasty. Hope there are only fixes.